app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js

```diff
@@ -33,7 +34,7 @@ const MigrationCompletedRow = ({ migration }) => {
   } else if (days === 1) {
     elapsedTime = sprintf(__('1 day %s:%s'), hours, minutes);
   } else {
-    elapsedTime = sprintf(__('%s:%s'), hours, minutes);
+    elapsedTime = sprintf(__('%s:%s:%s'), hours, minutes, seconds);
```

Comment:
@AparnaKarve it is possible to do hh:mm:ss for alignment purposes?

Reply:
@serenamarie125 Sure. I have made the required changes and have updated the screenshot above.

Reply:
great thank you @AparnaKarve !